app/controllers/miq_task_controller.rb

```diff
@@ -376,7 +306,7 @@ def build_query_for_status(opts)
   end
 
   def build_query_for_queued
-    ["(#{db_table}state=? OR #{db_table}state=?)", %w(waiting_to_start Queued)]
+    ["(miq_tasks.state=? OR miq_tasks.state=?)", %w(Waiting_to_start Queued)]
```

Comment:
@yrudman I know this code has been there before. But in the future, these parts should be moved into
backend and UI should only call nice backend methods rather than constructing sqls like this.